### PR TITLE
DDF-4364 Update Country Converter Version

### PR DESCRIPTION
**Pull request by willwill96**
#### What does this PR do?
Updates the Country Code Converter Version to the most recent release
#### Who is reviewing it? 
@GabrielFabian 
@gjvera 
@mojogitoverhere 
@rymach 
#### Ask 2 committers to review/merge the PR and tag them here.
@bdeining 
@djblue 
#### How should this be tested?
Just needs a build and a successful install. 
#### What are the relevant tickets?
[DDF-4364](https://codice.atlassian.net/browse/DDF-4364)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -190,7 +190,7 @@
         <commons-pool.version>1.6</commons-pool.version>
         <commons-validator.version>1.6</commons-validator.version>
         <components-font-awesome.version>4.7.0</components-font-awesome.version>
-        <countryconverter.version>0.1.3</countryconverter.version>
```
Comment (mojogitoverhere) [inline pom.xml]: ❗️ 	I don't think we should use a snapshot version. `0.1.4` was released an hour ago

**Comment by willwill96** [comment]
@mojogitoverhere Fixed

**Comment by willwill96** [comment]
build now

**Comment by bdeining** [comment]
build now

**Comment by GabrielFabian** [comment]
hero build in progress

**Pull request merged**